src/bindgen/utilities.rs

```diff
@@ -116,6 +116,47 @@ pub trait SynAttributeHelpers {
             })
     }
 
+    fn find_deprecated_note(&self) -> Result<Option<String>, String> {
```

Comment:
I have no idea around where to put these utilities.

Reply:
This one looks great, thanks! But rather than aborting parsing the item when we fail to parse `#[deprecated]`, I'd rather make this return an `Option<String>`, and return `None` for the error cases (maybe `warn!` ing if you want).